src/controller/java/src/chip/devicecontroller/model/AttributeWriteRequest.java

```diff
@@ -23,9 +23,9 @@
 
 /** An attribute write request that should be used for interaction model write interaction. */
 public final class AttributeWriteRequest {
-  private final ChipPathId endpointId, clusterId, attributeId;
-  private final Optional<Integer> dataVersion;
-  private final byte[] tlv;
+  private ChipPathId endpointId, clusterId, attributeId;
+  private Optional<Integer> dataVersion;
+  private byte[] tlv;
```

Comment:
Was this change accidental? The fields aren't mutated.

Reply:
yes, accidental copy from the earlier stage of write pr...fixed, thanks